pkg/rehearse/jobs.go

```diff
@@ -671,9 +671,7 @@ func SelectJobsForChangedRegistry(regSteps []registry.Node, allPresubmits presub
 	}
 	// The order is INTENTIONALLY reversed to cheaply increase the chance of hitting
 	// a useful rehearsal (prefer higher OCP versions)
-	sort.Slice(sortedConfigs, func(i, j int) bool {
-		return sortedConfigs[i].Info.Filename > sortedConfigs[j].Info.Filename
-	})
+	sortedConfigs = SortConfigs(sortedConfigs)
```

Comment:
I would suggest to really only focus implementing the `less` function to be used in the sort. The resulting sort order is derivative from the `less` function, so there's no reason why to extract the `sort.Slice` too. The `less` function is also better testable - it's just two inputs and a bool.

```golang
func moreRelevant(one, two *config.DataWithInfo) bool {
...
}

func SelectJobsForChangedRegistry(...) {
        ...
	sort.Slice(sortedConfigs, func(i, j int) bool {
		return moreRelevant(sortedConfigs[i], sortedConfigs[j])
	})
```